src/topp/ConsensusID.cpp

```diff
@@ -792,10 +810,22 @@ class TOPPConsensusID :
           }
         }
       }
-      else
+      else // link spectra by RT and mz proximity and do ConsensusID on their PSMs
       {
+        if (in.size() != 1)
+        {
+          OPENMS_LOG_WARN << "ConsensusID on idXML without the --per_spectrum flag, expects a single idXML file."
+          "Please merge the files with IDMerger using its default settings." << std::endl;
+        }
+        // note: this requires a single merged idXML file.
         IdXMLFile().load(in[0], prot_ids, pep_ids, document_id);
 
+        if (prot_ids.size() == 1)
+        {
+          OPENMS_LOG_WARN << "ConsensusID on idXML without the --per_spectrum flag usually expects a merged idXML file"
+          "with multiple runs. Only one run found in the first file." << std::endl;
```

Comment:
Maybe add a line what this implies (e.g., no processing is done, no new scores are calculated whatever. )

Reply:
Good question. The featurelinker only maps between different maps right? (Not inside the same?) It is hard to predict but nothing should happen. But it will be converted to a FeatureMap and back etc so I bet at least minor things will change.

Reply:
Now I am just failing it. Would probably nonsense to run it. I hope no one abused it like this.